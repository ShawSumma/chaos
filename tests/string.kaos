str var7 = "string1"
var7 = 'string2'
print var7
string var8 = 'string3'
var8 = "string4"
print var8

str a = 'hey'
str b = a[1]
str c = 'bom'
c = a[2]
print b
print c
symbol_table

str k = 'chaos'
k[1] = 'k'
del k[0]
print k
