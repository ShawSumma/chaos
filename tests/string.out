string2
string4
e
y
[start] =>
	{name: var7, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: var8, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: a, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: b, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
	{name: c, 2nd_name: (null), key: (null), scope: main, depth: 0, type: 2, 2nd_type: 0, value_type: 3, role: 0, param_of: } =>
[end]
